src/Components/ResourcePicker/ResourcePicker.tsx

```diff
@@ -143,14 +143,15 @@ const ResourcePicker: React.FC<IResourcePickerProps> = ({
 
     const areValuesEqual = useCallback(
         (value1: string, value2: string) => {
+            if (!value1 || !value2) return false;
```

Comment:
I see you had to make the same changes to the EnvironmentPickerManager file as well. It seems like it would make sense to move this to a utils file so we don't end up with bugs in the future if we fix something in one of them and not in the other.

Reply:
fixed: https://github.com/microsoft/iot-cardboard-js/pull/684/commits/bd1b8e1e247bd78cff57d56956330d4e289fe77d#diff-d815c7801f566cf88c3c2129276d79d0ce9cf6c7b3f30519d2ffcff99b9e52b9R673